TinYard/Framework/API/Interfaces/IMapper.cs

```diff
@@ -10,13 +10,15 @@ public interface IMapper
         event Action<IMappingObject> OnValueMapped;
         IMappingFactory MappingFactory { get; }
 
-        IMappingObject Map<T>();
+        IMappingObject Map<T>(string mappingName = null);
```

Comment:
I'm not 100% on these functions getting an optional parameter.. I feel they may be better suited as overloads for better clarity